lib/ansible/modules/network/asa/asa_command.py

```diff
@@ -68,35 +68,37 @@
 """
 
 EXAMPLES = """
-# Note: examples below use the following provider dict to handle
+# Note: examples below use the following group_vars/asa.yaml file to handle
 #       transport and authentication to the node.
 ---
-vars:
-  cli:
-    host: "{{ inventory_hostname }}"
-    username: cisco
-    password: cisco
-    authorize: yes
-    auth_pass: cisco
-    transport: cli
+ansible_connection: network_cli
+ansible_network_os: asa
+ansible_user: ansible
+ansible_become: yes
+ansible_become_method: enable
+ansible_become_pass: cisco
+command_timeout: 60
+
 
 ---
-- asa_command:
+- name: "Show the ASA version"
+  asa_command:
     commands:
       - show version
-    provider: "{{ cli }}"
 
-- asa_command:
+- name: "Show ASA drops and memory"
+  asa_command:
     commands:
       - show asp drop
```

Comment:
Is `asp` correct? It's not new content, so not really part of the PR, but to me this looks like a typo - should it be `asa`?